connect-example/src/main/java/com/stripe/android/connect/example/ui/appearance/AppearanceView.kt

```diff
@@ -38,7 +35,7 @@ fun AppearanceView(
         navigationIcon = {
             IconButton(onClick = onDismiss) {
                 Icon(
-                    imageVector = Icons.AutoMirrored.Default.ArrowBack,
+                    imageVector = Icons.Default.Close,
```

Comment:
👀 More standard to use `X` for a modal

Reply:
Makes sense. In that case can we change SettingsView to match?

Reply:
Maybe, but SettingsView isn't a modal, which can/should be a back arrow

Reply:
Hmmm, I see what you're saying. I'm fine keeping it as-is then